_includes/footer-menu.html

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-3">
       <ul class="footer-secondary-nav nav nav-pills nav-justified">
-        {{ site.data.footer-menu|build }}
+        {{ site.data.main-menu|build }}
```

Comment:
@richmahn - as far the main menu being flat, it is already hierarchical.  I'm Putting additional questions into https://github.com/unfoldingWord-dev/door43.org/issues/67 (that already has a question neither you or Jesse responded to).